vm/runtime.ts

```diff
@@ -2146,3 +2290,8 @@ var TN = J2ME.throwNegativeArraySizeException;
 
 var PE = J2ME.preempt;
 var PS = 0; // Preemption samples.
+
+var OBJ_HDR_SIZE = J2ME.Constants.OBJ_HDR_SIZE;
```

Comment:
> Now that I look around I don't see these used in JS, so let's just remove them from the globals.

That's true for *HASH_CODE_OFFSET* too, so I've removed both globals (and made *emitKlass* use the actual *OBJ_HDR_SIZE* value) in a63369d.